_locales/it/messages.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": {
+    "message": "Tabs aside",
+    "description": "name of the extension. Displayed in the manifest and pane header"
+  },
+  "description": {
+    "message": "Il classico menu di Microsoft Edge \"Tabs Aside\"",
+    "description": "Extension description"
+  },
+  "author": {
+    "message": "Michael \"XFox\" Gordeev",
+    "description": "Author name"
+  },
+  "options": {
+    "message": "Opzioni",
+    "description": "Alternative text for options button in the pane"
+  },
+  "closePanel": {
+    "message": "Chiudi",
+    "description": "Alternative text for close panel button"
+  },
+  "loadOnRestore": {
+    "message": "Riapri le schede al ripristino",
+    "description": "Label for option"
+  },
+  "showDeleteDialog": {
+    "message": "Mostra una finestra di conferma prima di eliminare qualcosa",
+    "description": "Label for option"
+  },
+  "swapIconAction": {
+    "message": "Sposta le finestre in Tabs Aside al click sull'estensione (%TOGGLE_SHORTCUT% o click con il tasto destro sull'icona dell'estensione per aprire il pannello)",
+    "description": "Label for option"
+  },
+  "github": {
+    "message": "Visita la pagina di GitHub",
+    "description": "Link title"
+  },
+  "contributors": {
```

Comment:
In version 2.0 this menu item will be replaced with "Changelog" menu item which will lead to GitHub Releases. 

So this:
```json
"contributors":
{
    "message": "Project contributors",
    "description": "Link title"
},
```
Must be replaced with this (with appropriate translation, of course):
```json
"changelog":
{
    "message": "Changelog",
    "description": "Link title"
},
```

Reply:
I'll resolve it in the next commit.